json/object.go

```diff
@@ -0,0 +1,31 @@
+package json
+
+import (
+	ej "encoding/json"
+	"reflect"
+)
+
+// Object is a general structure to represent JavaScript objects.
```

Comment:
```suggestion
// Object is a general structure to represent JSON objects.
```
I think `represent JavaScript object` sounds misleading.

Reply:
Thank you!